### PR TITLE
Update with missing default Kartographer params

### PR DESCRIPTION
**Pull request by ugochimobi**
please review and merge appropriately

### CODE REVIEW

**Comment by ugochimobi** [comment]
Oh, I never thought of that

**Comment by ugochimobi** [comment]
Although, I doubt it

File: LocalSettings.php [hunk 1]
```diff
@@ -1922,12 +1922,40 @@
...
+		'gratispaideiawiki' => [
+			'osm-intl',
+			'osm',
+		],
+		'gratisdatawiki' => [
+			'osm-intl',
+			'osm',
+		],
+	],
+	'wgKartographerEnableMapFrame' => [
```
[the remark this answers is not in the document]
Comment (ugochimobi) [reply LocalSettings.php:1948]: Actually the wgKartographerSrcsetScales is not defautly set

[the remark this answers is not in the document]
Comment (ugochimobi) [reply LocalSettings.php:1948]: Most of them aren't default configs

**Comment by RhinosF1** [comment]
They are according to the docs

**Comment by ugochimobi** [comment]
The docs on mediawiki?

**Comment by RhinosF1** [comment]
Yes

File: LocalSettings.php [hunk 2]
```diff
@@ -1922,12 +1922,40 @@
...
+		'gratispaideiawiki' => [
+			'osm-intl',
+		],
+		'gratisdatawiki' => [
+			'osm-intl',
+		],
 	],
 	'wgKartographerSrcsetScales' => [
 		'default' => false,
+		'gratispaideiawiki' => [1.3, 1.5, 2, 2.6, 3],
```
Comment (Reception123) [inline LocalSettings.php:1934]: please use different lines like [
1.3,
1.5,

**Review by Reception123 — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 2 above)
Comment (ugochimobi) [reply LocalSettings.php]: Done

**Comment by ugochimobi** [comment]
Seems these configurations are defaulting from extension.json

**Pull request closed without merging**